File: portal/src/models/PlayerDto.ts
export interface PlayerDto {
    id: number;
    name: string;
    email: string;
    auth0UserId: string | null;
    points: number;
    wins: number;
    isAdmin: boolean;
}

export const getDefaultPlayerDto: () => PlayerDto = () => ({
    id: -Math.random(),
    name: "",
    email: "",
    auth0UserId: null,
    points: 0,
    wins: 0,
    isAdmin: false
});

File: api/Oc6.Bold/Dtos/PlayerDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Linq.Expressions;
using System.Text;
using System.Threading.Tasks;

namespace Oc6.Bold.Dtos
{
    public record PlayerDto(int Id, string Name, string Email, string? Auth0UserId, bool IsAdmin, int Points, int Wins);
}
